### ISSUE TITLE
User Token

### ISSUE THREAD

**Issue by osu** [Jisagi/Discord-guild-copy#55]
Is it possible to use a discord user token beside using a bot token ? 

I actually need to copy a discord that isn't mine so I can't put a bot on it, so need to use an account to copy it and use a bot to paste it.

**Comment by osu** [comment]
I keep getting a Error [TOKEN_INVALID] when trying to use a user token and not bot token (works fine with bot token)

**Comment by Jisagi** [comment]
TL;DR: No, doesn't work and even if, they'd probably ban you for using it.

Long version:
1. The discord.js (d.js) library doesn't even support user tokens anymore, which means, they just don't work at all, even if you try. In theory my script should forbid the use of a user token, but it fails to login even before my script is able to check it, because d.js just doesn't know how to handle user tokens (anymore)
2. Older versions of d.js would support it, BUT those don't support the stuff I do with the script, so they don't work either
3. Even if everything would work with a user token, discord would most likely ban you for doing it. The script does a lot of stuff which already triggers ratelimits with a bot token. Those limit are probably even harsher for user token and therfore pretty easy catchable for them. They already banned selfbots, because those use the user token and it's api endpoint.
